src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -241,16 +241,23 @@ public void initializeFromEntity(@NotNull final EntityCitizen entity)
      */
     private String generateName(@NotNull final Random rand)
     {
-        final String firstName;
-        if (!female)
+        String citizenName;
+        if (female)
         {
-            firstName = getRandomElement(rand, Configurations.maleFirstNames);
+            citizenName = String.format("%s %s. %s", getRandomElement(rand, Configurations.femaleFirstNames), getRandomLetter(rand), getRandomElement(rand, Configurations.lastNames));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)